go/worker/registration/worker.go

```diff
@@ -354,19 +380,14 @@ func (w *Worker) consensusValidatorHook(n *node.Node) error {
 		}
 	}
 
+	// XXX: Could use registry.VerifyAddress() here to filter out invalid addresses.
```

Comment:
Good idea to prevent submitting invalid registrations.

Reply:
done